### ISSUE TITLE
Dirty Duck Tutorial: semantic.events not found

### ISSUE THREAD

**Issue by rthawkins** [dssg/triage#736]
```sql
select
  event_id
  entity_id,
  date,
  zip_code,
  type
  from
      semantic.events
 where random() < 0.001
 limit 5;
```

When attempting to perform the above tutorial query, it returns the following message: "relation "semantic.events" does not exist"

OS: WIndows 10 Home x64

**Comment by nanounanue** [comment]
Thank you for reporting this.

Did you run `tutorial.sh up`? Maybe is related to a bug in the `tutorial.sh` script, is being fixed in #746

**Comment by nanounanue** [comment]
Closed with #746
